### ISSUE TITLE
Add Chinese translation

### ISSUE THREAD

**Issue by skiselkov** [skiselkov/BetterPushbackC#51]
Need a translation of text and voice messages into Chinese.

The text messages can be found here: https://github.com/skiselkov/BetterPushbackC/blob/master/data/po/strings.pot

Additional texts that need to be translated are on the icons:
* "Accept plan"
* "Cancel plan"
* "Connect first"
* "Delete segment"
* " Click & Drag"
* "Move view"
* "Click"
* "Place segment"
* "Mouse wheel"
* "Rotate segment"

The text of the voice messages in English is:
* `connected.wav`: "Tow connected and bypass pin inserted. Release parking brake."
* `disco.wav`: "Disconnecting tow. Stand by."
* `done_left.wav`: "Tow is disconnected and bypass pin has been removed, hand signal on the left, we'll see you next time and have a safe flight."
* `done_right.wav`: "Tow is disconnected and bypass pin has been removed, hand signal on the right, we'll see you next time and have a safe flight."
* `driving_up.wav`: "Ground to cockpit. Tow is driving up."
* `op_complete.wav`: "Operation complete. Set parking brake."
* `plan_end.wav`: "Ground to cockpit. Plan acknowledged, call me through the menu when you are ready."
* `plan_start.wav`: "Ground to cockpit. Please show me where you want to go."
* `ready2conn.wav`: "Ok, all doors and hatches are closed, ready to connect. Set parking brake."
* `start_pb.wav`: "Starting pushback and you may start engines."
* `start_tow.wav`: "Starting tow and you may start engines."
* `winch.wav`: "Winching strap and adapter in position. Release parking brake when ready to start pushback."

The voice messages need to be recorded *as cleanly as possible*. If you are unfamiliar with Git, you can place the translations and audio files into a ZIP file and attach them to a comment here. Any help is greatly appreciated.

**Comment by skiselkov** [comment]
@cpuwolf Please test Chinese strings & icons on v0.35. You can select the language via the plugins -> BetterPushback -> Preferences window.

**Comment by cpuwolf** [comment]
icons are correct, but menu is not quite right. 

![x-plane 08 05 2017 - 09 59 04 05](https://user-images.githubusercontent.com/1320329/28992051-2bd78054-79c5-11e7-99ef-58ad7a4d28b4.png)
![x-plane 08 05 2017 - 09 57 57 01](https://user-images.githubusercontent.com/1320329/28992052-31496746-79c5-11e7-9d51-2701707ca943.png)

**Comment by cpuwolf** [comment]
the string should be

msgid "Pre-plan pushback"
msgstr "开始设计推车计划"

msgid "Close pushback planner"
msgstr "关闭推车计划"

but you see chinese string above, it doesnt match. do you have wrong calculation on UTF-8 string length?

**Comment by skiselkov** [comment]
@cpuwolf Sorry about that. I believe I merged your commit as-is, but maybe I messed something up afterwards.

**Comment by skiselkov** [comment]
Alright, so I've figured it out. My string parser and X-Plane's UTF-8 processing are fine. The problem is in the default proportional font, which is missing simplified Chinese characters. When I switch the characters around to traditional ones, everything looks ok. Presumably they just never bothered to use a font which includes simplified Chinese characters, because they only support Japanese natively (which uses traditional Chinese characters).

**Comment by cpuwolf** [comment]
hi Skiselkov,
per my understanding, simplified Chinese is subset of tranditional Chinese. it will be good. Thank you for your quick response

**Comment by skiselkov** [comment]
Ok. Can you please translate them to traditional script for me? Or is google translate good enough not to break anything? Also, can you please also translate these for me:
* "Preferences..."
* "BetterPushback Preferences"
* "User interface"
* "X-Plane's language"
* "Ground crew audio"
* "My language only at domestic airports"
* "My language at all airports"
* "English at all airports"
* "Save preferences"
* "Ground crew speaks my language only if the country the airport is in speaks my language. Otherwise the ground crew speaks English with a local accent."
* "Ground crew speaks my language irrespective of what country the airport is in."
* "Ground crew always speaks English with a local accent."
* "Save current preferences to disk."

**Comment by cpuwolf** [comment]
yes sure. could you have a look my pull request?

**Comment by cpuwolf** [comment]
1. "Preferences..."
   “设置...”
2. “BetterPushback Preferences”
   "BetterPushback设置"
3. "User interface"
   “用户界面”
4. "X-Plane's language"   
   “X-Plane系统语言”
5. "Ground crew audio"
   “地勤人员声音”
6. "My language only at domestic airports"
   “我的语言只在国内机场使用”
7. "My language at all airports"
   “我的语言在所有机场使用”
8. "English at all airports"
   “所有机场用英文”
9  "Save preferences"
   “保存首选项”
10."Ground crew speaks my language only if the country the airport is in speaks my language. Otherwise the ground crew speaks English with a local accent."
   “地面人员只有在使用我的语言的国家的机场时，才会说我的语言，否则地勤人员会说当地口音英语。”
11."Ground crew speaks my language irrespective of what country the airport is in."
   “无论机场在哪个国家，地勤人员都会说我的语言。”
12."Ground crew always speaks English with a local accent."
   “地勤人员总是说当地口音英语。”
13."Save current preferences to disk."   
   “保存当前设置”。

**Comment by skiselkov** [comment]
@cpuwolf I'm looking at `done_left.mp3` and `done_right.mp3` and they're the same file. Can you please check.

**Comment by skiselkov** [comment]
Switched all text to traditional Chinese characters to fix the X-Plane incompatibility. Please test v0.38.

**Comment by cpuwolf** [comment]
test done! all good! and your modified chinese icons are better, which has bigger charators! Great

**Comment by TheAutumnSky** [comment]
Hi Saso. Today I'll try to ask a friend to record audio phrases in Chinese. It will be a Russian person, so there will be an accent. I hope for the first time will help you!

**Comment by skiselkov** [comment]
@TheAutumnSky I don't think you need to record another audio set, I can use the one from @cpuwolf, which will be coming out with the next build tonight.

**Comment by skiselkov** [comment]
@cpuwolf Chinese audio has been integrated into build 0.39. Please test if it sounds ok.

**Comment by skiselkov** [comment]
I'd like to request a few more text-only translations for a new feature I'm working on:
* icons:
    * "Reconnect"
    * "Disconnect & hand signals"
* texts:
    * "Disconnect tow + headset and switch to hand signals."
    * "Reconnect tow and await further instructions."

**Comment by Jzzzzth** [comment]
just a suggestion, the chinese sound files feel a bit unnatural and it give a feeling of google translates.
but this might not be a high priority for you guys XD, and it could work as it is.

**Comment by skiselkov** [comment]
@Jzzzzth I'm not Chinese myself, so I rely on contributions. If you would be willing record a better sound set, I'd love to integrate it.

**Comment by cpuwolf** [comment]
icons:

"Reconnect"
"重新连接"
"Disconnect & hand signals"
"断开连接并看手势"

texts:

"Disconnect tow + headset and switch to hand signals."
"断开拖杆和耳机然后注意手势"
"Reconnect tow and await further instructions."
"重新连接拖杆，等待指令"
"Preferences..."
"设置..."
"BetterPushback Preferences"
"BetterPushback设置"
"User interface"
"界面设置"
"X-Plane's language"
"X-Plane系统语言"
"Ground crew audio"
"地勤人员声音"
"My language only at domestic airports"
"只在国内机场说中文"
"My language at all airports"
"所有机场说中文"
"English at all airports"
"所有机场说英文"
"Save preferences"
"保存首选项"
"Ground crew speaks my language only if the country the airport is in speaks my language. Otherwise the ground crew speaks English with a local accent."
"地面人员只有在说中文的国家的机场时，才会说中文，否则地勤人员会说当地口音英语"
"Ground crew speaks my language irrespective of what country the airport is in."
"无论机场在哪个国家，地勤人员都会说中文"
"Ground crew always speaks English with a local accent."
"地勤人员总是说当地口音英语"
"Save current preferences to disk."
"保存当前设置"

**Comment by cpuwolf** [comment]
icons:

"Reconnect"
"重新連接"
"Disconnect & hand signals"
"斷開連接並看手勢"

texts:

"Disconnect tow + headset and switch to hand signals."
"斷開拖桿和耳機然後註意手勢"
"Reconnect tow and await further instructions."
"重新連接拖桿，等待指令"
"Preferences..."
"設置..."
"BetterPushback Preferences"
"BetterPushback設置"
"User interface"
"界面設置"
"X-Plane's language"
"X-Plane系統語言"
"Ground crew audio"
"地勤人員聲音"
"My language only at domestic airports"
"只在國內機場說中文"
"My language at all airports"
"所有機場說中文"
"English at all airports"
"所有機場說英文"
"Save preferences"
"保存首選項"
"Ground crew speaks my language only if the country the airport is in speaks my language. Otherwise the ground crew speaks English with a local accent."
"地面人員只有在說中文的國家的機場時，才會說中文，否則地勤人員會說當地口音英語"
"Ground crew speaks my language irrespective of what country the airport is in."
"無論機場在哪個國家，地勤人員都會說中文"
"Ground crew always speaks English with a local accent."
"地勤人員總是說當地口音英語"
"Save current preferences to disk."
"保存當前設置"

**Comment by cpuwolf** [comment]
@Jzzzzth right now is the voice is translated from text by machine, yes you are right.  skiselkov is updating everyday. high possibility  is will request more audio files in the future.  so keep current machine speak temporally.  I would love to record my voice if you like in the future.

**Comment by Jzzzzth** [comment]
@cpuwolf no problem at all, and i love this plugin. i tried to record some voice lines myself but my devices arent good enough for that and im not working in the industry so i am not certain with my translation so i didnt finish the recording. but yes it can be improved, but maybe after everything else major got sorted first
